### PR TITLE
chore: Create empty spark-icons module

### PR DESCRIPTION
**Pull request by soulcramer**
## 🧑‍ Changes description

Add a new :spark-icons module  and make :spark depend from it

## 🤔 Context

It will contains our SparkIcon class with the drawables used by the design system

## 👋 Other info

Initialization of #117

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -30,7 +31,9 @@ airbnb-showkase = { module = "com.airbnb.android:showkase", version.ref = "showk
 airbnb-showkase-processor = { module = "com.airbnb.android:showkase-processor", version.ref = "showkase" }
 
 android-documentation-plugin = { module = "org.jetbrains.dokka:android-documentation-plugin", version.ref = "androidDocumentationPlugin" }
+# Artifact: https://mvnrepository.com/artifact/androidx.appcompat/appcompat
```
Comment (SimonMarquis) [inline gradle/libs.versions.toml:34]: IMO this info is useless. We should either redirect to a public github project or changelog. Also, IntelliJ now directly shows the available updates.

File: spark-icons/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    lint {
+        warningsAsErrors = true
+        sarifReport = true
+    }
+}
+
+dependencies {
+    implementation(platform(libs.androidx.compose.bom))
+    implementation(libs.androidx.compose.ui)
+    api(libs.androidx.appCompat.ressources) // Needed for compat vector drawables
```
Comment (SimonMarquis) [inline spark-icons/build.gradle.kts:78]: Does it needs to be exposed as an api though?

File: spark-icons/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (SimonMarquis) [inline spark-icons/src/main/AndroidManifest.xml:1]: Could be removed I think

Comment (soulcramer) [reply spark-icons/src/main/AndroidManifest.xml]: Done in c995245

File: spark-icons/build.gradle.kts (hunk 2 above)
Comment (soulcramer) [reply spark-icons/build.gradle.kts]: I'm not sure but we can change it back to api later if needed 👍 
c995245

File: gradle/libs.versions.toml (hunk 1 above)
Comment (soulcramer) [reply gradle/libs.versions.toml]: I personally use androidx.tech to get the changelog & the sources is it good enough or should we reference the androidx Github?
c995245

**Review by SimonMarquis — APPROVED** [review]

**Pull request merged**